ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -72,6 +72,30 @@ <h3 class="title">
         <mat-icon>delete</mat-icon>
         <span>DROP INDEX</span>
       </button>
+      <button
```

Comment:
On adding a table `x@1` in source database and starting the migration the table names look erroneous with `x@1` shown as dropped : https://drive.google.com/file/d/1fL8y2B7PgZgt2o6R5IQchkZK0PK9xfyW/view?usp=sharing&resourcekey=0-Re3eSUcuHMA7_sLYag0myA

Reply:
Fixed

Reply:
Dropping and restoring this table is still not working. Please test for all the cases with special character in table name , column name, etc

Reply:
The bug is in set-parent-table API and it is fix temporarily. With special character in source name, the pre-existing api are failing in some instance because the existing API are base on table-name. Once they are converted to Ids during name->id refactoring, then these special characters issues will be fixed properly.